ServerCore/Components/Layout/MainLayout.razor

```diff
@@ -0,0 +1,15 @@
+﻿@inherits LayoutComponentBase
```

Comment:
Is this .net 8.0 boilerplate or something else? If it's boilerplate, consider making a PR to migrate to 8.0 and then having a subsequent PR for the syncing backend. If something else, can you document what it is for?

Reply:
.NET 8.0 Blazor boilerplate